libs/map/src/lib/components/layer/wmts-layer/vl-map-wmts-layer.ts

```diff
@@ -82,17 +82,24 @@ export class VlMapWmtsLayer extends VlMapLayer {
         return 'image/png';
     }
 
-    get __grbMatrixSet() {
-        return 'BPL72VL';
+    get __grbMatrixSet(): string {
+        return this.getAttribute('matrix-set') || "BPL72VL";
+    }
+
+    get __prefixMatrix(): boolean {
+        return this.getAttribute('matrix-prefix') != undefined;
```

Comment:
ik denk dat het in dit geval niet uitmaakt, maar typisch gaan we standaard eerder met `!==` / `===` werken ipv `!=` / `==`

Reply:
Ja ik zag elders ook `==` vandaar, ik zal het aanpassen, ben akkoord :-)